lib/ripper-tags/default_formatter.rb

```diff
@@ -27,6 +37,16 @@ def extra_flag?(flag)
       options.extra_flags && options.extra_flags.include?(flag)
     end
 
+    def supported_fields() [] end
+
+    def field?(field)
+      options.fields && options.fields.include?(field)
+    end
+
+    def excmd?(excmd)
```

Comment:
This helper is not really necessary. The caller can simply do `options.excmd == 'number'`